auth_style/templates/allauth/elements/img.html

```diff
@@ -1 +1 @@
-<img src="{{ attrs.src }}" {% if attrs.alt %}alt="{{ attrs.alt }}"{% endif %}>
+<img class="rounded-lg dark:bg-[rgba(255,255,255,.2)]" src="{{ attrs.src }}" {% if attrs.alt %}alt="{{ attrs.alt }}"{% endif %}>
```

Comment:
Likewise less magical here too.

Reply:
I'm going a lot lighter than previously to ensure consistent usability of the QR code.